File: MODUL 2 PYTHON/PRAK110-2210817310004-Muhammad Adika Rizqullah.py
import math
sisiA, sisiC = 12, 5
sisiB = sisiA**2 + sisiC**2
sisiB = round(math.sqrt(sisiB))
kelilingSegitiga = sisiA + sisiB +sisiC 
luasSegitiga = 0.5 * sisiA * sisiC 
print("Diketahui :")
print("Alas =", sisiC, "cm")
print("Tinggi =", sisiA, "cm\n")
print("Jawab :")
print("Sisi A =", sisiA, "cm")
print("Sisi B =", sisiB, "cm")
print("Sisi C =", sisiC, "cm")
print("Keliling =", kelilingSegitiga, "cm")
print("Luas =", round(luasSegitiga), "cm")

File: MODUL 2 C/PRAK110-2210817310004-Muhammad Adika Rizqullah.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

void main (){
    int alas = 5, tinggi = 12, A, B, C, K, L, P;
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
        A = tinggi;
        printf("Sisi A = %d cm\n", A);
        C = alas;
        printf("Sisi C = %d cm\n", C);
        P = (A * A) + (C * C);
B = sqrt(P);
        printf("Sisi B = %d cm\n", B);

        K = A + B + C;
        printf("Keliling = %d cm \n", K);
        L = 0.5 * A * C;
        printf("Luas = %d cm\n", L);
}
